tests/test_creation/test_geo_features.py

```diff
@@ -0,0 +1,266 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from feature_engine.creation import GeoDistanceTransformer
+
+
+class TestGeoDistanceTransformer:
```

Comment:
What's the advantage of creating a class? 

We'd normally have the tests individually, ie, outside the class.